soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4055,27 +4069,42 @@ void DrawRandoEditor(bool& open) {
 
                     // Cuccos to return
                     SohImGui::EnhancementSliderInt("Cuccos to return: %d", "##RandoCuccosToReturn",
-                                                    "gRandomizeCuccosToReturn", 0, 7, "", 7);
+                                                    "gRandomizeCuccosToReturn", 0, 7, "", 7, true);
                     InsertHelpHoverText(
                         "The amount of cuccos needed to claim the reward from Anju the cucco lady"
                     );
                     ImGui::Separator();
 
                     // Big Poe Target Count
                     SohImGui::EnhancementSliderInt("Big Poe Target Count: %d", "##RandoBigPoeTargetCount",
-                                                    "gRandomizeBigPoeTargetCount", 1, 10, "", 10);
+                                                    "gRandomizeBigPoeTargetCount", 1, 10, "", 10, true);
                     InsertHelpHoverText(
                         "The Poe collector will give a reward for turning in this many Big Poes."
                     );
                     ImGui::Separator();
 
                     // Skip child stealth
-                    SohImGui::EnhancementCheckbox(Settings::SkipChildStealth.GetName().c_str(), "gRandomizeSkipChildStealth");
-                    InsertHelpHoverText(
-                        "The crawlspace into Hyrule Castle goes straight to Zelda, skipping\n"
-                        "the guards."
-                    );
+                    // Disabled when Skip Child Zelda is active
+                    if (disableZeldaRelatedOptions) {
+                        CVar_SetS32("gRandomizeSkipChildStealth", 0);
+                        // Also disable Weird Egg because it's on a different tab and wouldn't be updated otherwise
+                        CVar_SetS32("gRandomizeShuffleWeirdEgg", 0);
+                    }
+                    ImGui::PushItemFlag(ImGuiItemFlags_Disabled, disableZeldaRelatedOptions);
+                    ImGui::PushStyleVar(ImGuiStyleVar_Alpha,
+                                        ImGui::GetStyle().Alpha * disableZeldaRelatedOptions ? 0.5f : 1.0f);
```

Comment:
nitpick here, and people may disagree with my stylistically here, but to me this would read more clearly as 
```cpp
ImGui::GetStyle().Alpha * (disableZeldaRelatedOptions ? 0.5f : 1.0f)
```

because it wasn't immediately clear to me from reading that the logic wasn't

```cpp
(ImGui::GetStyle().Alpha * disableZeldaRelatedOptions) ? 0.5f : 1.0f
```

this applies to all the other alpha setting spots as well

Reply:
I copied it from the existing one that was used for disabling all rando settings. But if it makes it more readable, I'll change all of them (including the existing one).